### PR TITLE
Fixes typo in bulwark description

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

ermm
## Why It's Good For The Game
I think github might get mad
## Changelog
:cl:
spellcheck: Fixed a typo in bulwark description
/:cl:

### CODE REVIEW

**Comment by Arctrooper8** [comment]
How is this a typo? I'm aware the word can be seen as a slur but it's also an actual word commonly used in terms of weakness' in armor. At least find a better replacement than "chunks" use gaps or holes.

**Comment by TiviPlus** [comment]
chink
1 of 5
noun (1)
ˈchiŋk 
Synonyms of chink
1
: a small cleft, slit, or fissure
a chink in the fence
2
: a weak spot that may leave one vulnerable
his lawyers found a chink in the law
3
: a narrow beam of light shining through a chink

**Comment by Lumipharon** [comment]
This is the correct usage. a 'chink in the armour' is a specific phrase/term lol, I unno why anyone would be offended.

**Comment by TiviPlus** [comment]
github nuked tg for using the f word for british meatballs so may as well be safe

File: code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm [hunk 1]
```diff
@@ -448,7 +448,7 @@
 /datum/action/ability/xeno_action/bulwark
 	name = "Royal Bulwark"
 	action_icon_state = "bulwark"
-	desc = "Creates a field of defensive energy, filling chinks in the armor of nearby sisters, making them more resilient."
+	desc = "Creates a field of defensive energy, filling chunks in the armor of nearby sisters, making them more resilient."
```
Comment (ivanmixo) [inline code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm:451]:
```suggestion
	desc = "Creates a field of defensive energy, filling gaps in the armor of nearby sisters, making them more resilient."
```

**Pull request merged**